django/db/models/fields/related.py

```diff
@@ -2064,6 +2064,15 @@ def db_type(self, connection):
     def db_parameters(self, connection):
         return {"type": self.db_type(connection), "check": []}
 
+    def from_db_value(self, value, connection, context):
```

Comment:
This should be done in `get_db_converters` not as `from_db_value` for performance reasons. Where possible, we should avoid the existence of any converter functions, even when they become noops like this.